hypervisor/arch/x86/guest/instr_emul_wrapper.c

```diff
@@ -44,8 +44,10 @@ int vm_get_register(struct vcpu *vcpu, enum cpu_reg_name reg, uint64_t *retval)
 		uint32_t field = get_vmcs_field(reg);
 
 		if (field != VMX_INVALID_VMCS_FIELD) {
-			if (reg <= CPU_REG_PDPTE3) {
+			if (reg <= CPU_REG_RFLAGS) {
```

Comment:
Better to define a CPU_REG_MAX in the enum, and use this MACRO here. Otherwise, readers doesn't know why it is CPU_REG_RFLAGS.

Reply:
Can we define CPU_REG_NATURAL_LAST and CPU_REG_64BIT_LAST for these condition statement and add some code comments? CPU_REG_NATURAL_LAST indicates the register name (less than CPU_REG_NATURAL_LAST) corresponds to natural width field in VMCS; CPU_REG_64BIT_LAST indicates the register name (less than CPU_REG_64BIT_LAST) corresponds to 64-bit width field in VMCS.